### ISSUE TITLE
Explain CHE_DOCKER_IP_EXTERNAL in "hosting" segment of docs

### ISSUE THREAD

**Issue by galgalesh** [eclipse/che-docs#75]
[The hosting segment of the docs](https://www.eclipse.org/che/docs/setup/getting-started/#hosting) explains that `CHE_HOST` has to be set to the correct value when hosting Che at a cloud service like DigitalOcean. This section should also mention that `CHE_DOCKER_IP_EXTERNAL` has to be changed too.

Setting `CHE_HOST` gets you a working che dashboard but your browser won't be able to connect to the workspaces unless you also set `CHE_DOCKER_IP_EXTERNAL`. Otherwise, your browser will try to contact the workspace on its internal address (172.17...).

**Comment by TylerJewell** [comment]
@galgalesh - setting this property should not be required for hosting if you have set CHE_HOST.  We had an error in 5.0.0-SNAPSHOT that should have fixed this in 5.1.0.  can you confirm the version that you are using?

But `CHE_DOCKER_IP_EXTERNAL` should only be set in selective situations, nad we document this in the `che.env` in the networking section of the file.

**Comment by galgalesh** [comment]
```
Your CLI version is '5.1.0'.
Your installed version is '5.1.0'.
```

**Comment by TylerJewell** [comment]
Are you running in a VM at DO?  Can you show us the start command that you used when starting Che.

**Comment by galgalesh** [comment]
I'm running it in a private cloud.

```bash
docker run -it --rm -v /var/run/docker.sock:/var/run/docker.sock -e CHE_HOST=172.28.0.34 -e CHE_PORT=8081 -v /home/ubuntu/:/data eclipse/che start
```

che.env:
```
CHE_HOST=172.28.0.34

#...

CHE_DOCKER_IP_EXTERNAL=172.28.0.34
```

When I comment the `CHE_DOCKER_IP` line, I can't connect to the workspaces anymore.

**Comment by TylerJewell** [comment]
thanks, @galgalesh

**Comment by TylerJewell** [comment]
@galgalesh - it looks like I was wrong and that the error was found in 5.0.0 but the fix did not make it into 5.1.0.  The PR for 3686 is open and solves this.
